Cql/Cql.CqlToElm/CqlLibraryString.cs

```diff
@@ -36,14 +36,15 @@ public static CqlLibraryString FromIdentifierAndString(
     /// </summary>
     /// <returns>A <see cref="Regex"/> to extract the library name and version.</returns>
     [GeneratedRegex("""
-                        (\s+\r?\n?)*        # Skip whitespace and newlines
+                        ^\s*                # Skip whitespace and newlines
                         library             # until "library" is found
                         \s+
-                        (?<lib>\S+)         # The name of the library
+                        (?<lib>\S+)         # The name of the library                                      
                         (
-                          \s+
+                          \s+               # at least one space
                           version
-                          \s'(?<ver>[^']+)' # The version of the library between single quotes
+                          \s+               # at least one space
```

Comment:
FYI. This line appears to be pretty important. The NCQA LSC_Bulk, LSC_Details, and LSC_Reporting are not building without considering multiple spaces between library and version.